doc/onboarding.md

```diff
@@ -1,63 +1,45 @@
-# How to onboard a new node in testnet
+# Support a new XMTP testnet node operator
```

Comment:
I updated this doc to be the xmtpd dev side of onboarding a new node operator - whereas the Helm doc is the node operator side of the process. WDYT?

Reply:
this file is auto-generated from code, so I added your changes here https://github.com/xmtp/xmtpd/pull/825